examples/data-objects/codemirror/src/presence.ts

```diff
@@ -36,8 +36,8 @@ class PresenceManager extends EventEmitter {
         this.presenceKey = `presence-${runtime.id}`;
 
         runtime.on("signal", (message: IInboundSignalMessage, local: boolean) => {
-            // Only process presence keys that are not local while we are connected
-            if (message.type === this.presenceKey && !local && runtime.connected) {
+            // Only process presence keys that are not local while we are connected and have a non-null clientId
+            if (message.type === this.presenceKey && !local && runtime.connected && message.clientId) {
```

Comment:
`ISignalMessage.clientId` can be `null`. The only case for this is on room join, so I don't think presence needs to be emitted for that. Let me know if I'm wrong.

Reply:
Makes sense. I think the type check will also fail but better to be more conservative.